README.md

```diff
@@ -74,29 +69,27 @@ Clone this repository to your desired folder:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
 
 👤 **Author1**
 
 - GitHub: [@githubhandle](git@github.com:KAYEMBAJONATHAN/Hello_Microverse.git)
 - Twitter: [@twitterhandle]()
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/jonathan-kayemba-b56247236/)
```

Comment:
- [ OPTIONAL] :thinking: I think it would be better if you used a your true handles to make sure your visitor know that your links are working :factory_worker: